### PR TITLE
Cover pages refactoring

### PR DESCRIPTION
**Pull request by mustiwebp**
What the title says.

### CODE REVIEW

File: lib/screens/questionaire_screen/widgets/last_cover.dart
Comment (S1rDan) [inline lib/screens/questionaire_screen/widgets/last_cover.dart:1]: What is the point of this widget?
It seems redundant.

Comment (mustiwebp) [reply lib/screens/questionaire_screen/widgets/last_cover.dart:1]: It is the widget specially for the last page of the questionnaire. It is different from the other cover pages thus I created an own widget for that.

File: lib/screens/questionaire_screen/widgets/last_cover.dart [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                          ),
+                        ],
+                      ),
+                    ),
+                  ),
+                  const Positioned(
+                    top: -20,
+                    right: 20,
+                    child: PicosSvgIcon(
+                      assetName: 'assets/Tipp.svg',
```
Comment (S1rDan) [inline lib/screens/questionaire_screen/widgets/last_cover.dart:128]: The colors aren't rendered for this SVG.
Check this out: [https://stackoverflow.com/a/65033951](https://stackoverflow.com/a/65033951)
I've tried it. After preparing the SVG with this tool and removing the colorFilter from PicosSvgIcon, the Picture gets rendered correctly.
I would suggest to try this solution for all SVG's and watch what happens.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/screens/questionaire_screen/widgets/last_cover.dart
Comment (S1rDan) [reply lib/screens/questionaire_screen/widgets/last_cover.dart:1]: It just adds two more widgets (SizedBox and Container) to to the very same copy pasted Column. It doesn't do anything else.

**Review by S1rDan — CHANGES_REQUESTED** [review]
Almost done.

**Pull request merged**